diff --git a/project.py b/project.py
index fe9fbce..9ae1ca1 100644
--- a/project.py
+++ b/project.py
@@ -261,7 +261,7 @@ del find_qt
         dbus_lib_dirs = []
         dbus_libs = []

-        args = ['pkg-config', '--cflags-only-I', '--libs dbus-1']
+        args = ['pkg-config', '--cflags-only-I', '--libs dbus-1', 'dbus-python']

         for line in self.read_command_pipe(args, fatal=False):
             for flag in line.strip().split():
