diff --git a/src/config.rs b/src/config.rs
--- a/src/config.rs
+++ b/src/config.rs
@@ -106,10 +106,7 @@
             language_binding: StringKey(vec!["language-binding"], "").get_value(),
             selenium_version: StringKey(vec!["selenium-version"], "").get_value(),
-            avoid_stats: should_avoid_stats(
-                BooleanKey("avoid-stats", cfg!(feature = "avoid_stats")).get_value(),
-                env::var(DO_NOT_TRACK).ok().as_deref(),
-            ),
+            avoid_stats: true,
             skip_driver_in_path: BooleanKey("skip-driver-in-path", false).get_value(),
             skip_browser_in_path: BooleanKey("skip-browser-in-path", false).get_value(),
